normits_demand/models/forecasting/EDGE_growth/ticket_splits.py

```diff
@@ -0,0 +1,250 @@
+# -*- coding: utf-8 -*-
+"""
+Module to process ticket type splitting factors.
+
+Splits by tag flow and purpose exist, which need to be converted to splits by station pair.
+"""
+# Built-Ins
+import pickle
+
+# Third Party
+import pandas as pd
+import numpy as np
+import caf.toolkit as ctk
+
+# Local Imports
+# pylint: disable=import-error,wrong-import-position
+# Local imports here
+from normits_demand.models.forecasting.edge_growth import utils
+from normits_demand.models.forecasting import forecast_cnfg
+from normits_demand.utils import file_ops
+# pylint: enable=import-error,wrong-import-position
+
+# # # CONSTANTS # # #
+
+# # # CLASSES # # #
+
+# # # FUNCTIONS # # #
+def append_dist(row):
```

Comment:
It would be good for this to be converted into some kind of input file so it's more clear how the TAG flows are being categorised